packages/sdk/src/client.ts

```diff
@@ -736,6 +746,47 @@ export function SoundClient({
     return network.chainId
   }
 
+  function _validateEditionConfig(config: EditionConfig) {
+    const { editionMaxMintableLower, editionMaxMintableUpper, fundingRecipient } = config
+
+    if (!isAddress(fundingRecipient) || fundingRecipient === NULL_ADDRESS) {
+      throw new InvalidFundingRecipientError({ fundingRecipient })
+    }
+
+    if (editionMaxMintableLower > editionMaxMintableUpper) {
+      throw new InvalidEditionMaxMintableError({
+        editionMaxMintableLower,
+        editionMaxMintableUpper,
+      })
+    }
+  }
+
+  function _validateMintConfigs(mintConfigs: MintConfig[]) {
+    for (const mintConfig of mintConfigs) {
+      const { maxMintablePerAccount } = mintConfig
+      if (maxMintablePerAccount < 1) {
+        throw new InvalidMaxMintablePerAccountError({ maxMintablePerAccount })
+      }
+
+      if (mintConfig.mintType === 'RangeEdition') {
+        const { maxMintableLower, maxMintableUpper, startTime, cutoffTime, endTime } = mintConfig
+        if (maxMintableLower > maxMintableUpper) {
+          throw new InvalidMaxMintableError({ maxMintableLower, maxMintableUpper })
+        }
+        if (!(startTime < cutoffTime && cutoffTime < endTime)) {
+          throw new InvalidTimeValuesError({ startTime, cutoffTime, endTime })
+        }
+      }
+
+      if (mintConfig.mintType === 'MerkleDrop') {
+        const { merkleRoot } = mintConfig
+        if (merkleRoot === NULL_BYTES32 || merkleRoot.slice(0, 2) !== '0x' || merkleRoot.length !== 66) {
```

Comment:
can you add some reference about this .length !== 66 validation?

Reply:
https://github.com/soundxyz/sdk/pull/133/commits/be632a403956bc28087c8c31d29bbce606577d44